src/indigo/less/shame.less

```diff
@@ -370,3 +362,163 @@ small.empty-label {
 .kbc-version-icon.last {
   color: #96d130;
 }
+
+// moved from ModalDialog.less
+.kbc-table-browser .modal-dialog {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.kbc-table-browser .modal-content {
+  height: auto;
+  min-height: 100%;
+  border-radius: 0;
+}
+
+// moved from configurations/styles.less
+.drag-handle-pending {
+  cursor: not-allowed;
+}
+
+.drag-handle-disabled {
+  cursor: not-allowed;
+  opacity: 0.65;
+}
+
+.drag-handle {
+  cursor: move;
+}
+
+// po pruzkumu a diskuzi s autorem  - Najlosem - jsem dosel k zaveru ze tenhle styl neni potreba
+//.kbc-json-editor textarea {
+//  resize: both;
+//  width: 100%;
+//  min-height: 300px
+//}
+
+//Vyhazuju, nebylo to izolovany a naopak to bylo prestylovany
+// h2 je definovanej asi na milionu mistech
+//h2 {
+//  line-height: 32px;
+//  margin-bottom: 10px;
+//}
+
+//moved form AddConfigurationForm.less
+
+//tohle bude jednoduchy odstranit, je to hotfix na poprasenej bootstrap
+//Zobrazuje se prizalozeni novy konfigurace gdwr
+.modal-header.add-configuration-form {
+  padding-top: 20px;
+  padding-bottom: 15px;
+  min-height: 79px;
+}
+
+.modal-header.add-configuration-form .close {
+  padding-top: 9px;
+}
+
+.modal-header.add-configuration-form .kbc-header {
+  float: left;
+}
+
+//moved form SidebarVersions.less
+// namet na presun dalsi komponenty do indiga - neni to nic nez jinyho nez variace <List> nebo <Media>
+.kbc-sidebar-versions {
+  margin-top: 15px;
+  border-top: 1px solid #dddddd;
+  margin-bottom: 20px;
+}
+
+.kbc-sidebar-versions .version-info {
+  margin-bottom: 10px;
+}
+
+.kbc-sidebar-versions .versions-link {
+  padding-left: 28px;
+  margin-top: 10px;
+}
+
+.kbc-sidebar-versions .versions-link a {
+  color: #7e8490;
+}
+
+.kbc-sidebar-versions .td.versions-status {
+  width: 31px;
+}
+
+
+//moved form SidebarJobs.less
+// namet na presun dalsi komponenty do indiga - stejne jako kbc-sidebar-versions
+.kbc-sidebar-jobs .td.kbc-td-status {
+  width: 31px;
+}
+
+.kbc-main-sidebar div.kbc-sidebar-jobs {
+  margin-bottom: 25px;
+}
+
+.kbc-sidebar-jobs .jobs-link {
+  padding-left: 28px;
+  margin-top: 10px;
+}
+
+.kbc-sidebar-jobs .jobs-link a {
+  color: #7e8490;
+}
+
+
+//moved from AuthorizationModal.less
```

Comment:
@janmichek znamenta to ze `.kbc-authorization-modal` ide prec kompletne? (tam bol ten wtf fix)

Reply:
Ano, wtf fix uz je k nicemu 
![image](https://user-images.githubusercontent.com/15363559/41709745-4062ee0c-7544-11e8-8180-34430b42aaa7.png)

potom uz ta classa '.kbc-authorization-modal' bude k nicemu. Specifikovala jenom wtf fix

Reply:
![image](https://user-images.githubusercontent.com/15363559/41709853-8b8edc88-7544-11e8-9a7a-eb10e8c6a909.png)

Reply:
ok, potom ok :)